app/assets/javascripts/hyrax/editor/controlled_vocabulary.es6

```diff
@@ -114,9 +128,25 @@ export default class ControlledVocabulary extends FieldManager {
   // '_destroy' hidden parameter
   removeFromList( event ) {
       event.preventDefault()
+      // Changing behavior of the remove button to add a new field if the last field is removed
+      // Using querySelector to find elements with data-attribute="based_near"
+      const basedNearElements = document.querySelectorAll('[data-attribute="based_near"]');
+      const parentsArray = Array.from(basedNearElements).map(element => element.parentElement);
+      const nonHiddenElements = parentsArray.filter(element => element.style.display !== 'none');
+      const nonHiddenCount = nonHiddenElements.length;
+      if (nonHiddenCount < 2){
+        let $listing = $(event.target).closest(this.inputTypeClass).find(this.listClass)
+        let $activeField = $listing.children('li').last()
+        $listing.append(this.createNewField($activeField));
+        this.postRemovalAdjustment += 1;
+      }
       let field = $(event.target).parents(this.fieldWrapperClass)
-      field.find('[data-destroy]').val('true')
-      field.hide()
+      if (field.find('.has-existing-value').length > 0) {
```

Comment:
It might be helpful to add a comment before this `if` statement to describe what it is for (for handling values already saved to an existing work vs newly added values)

Reply:
Maybe i'm misunderstanding, but isn't the comment backwards? I thought it removed the field if it didn't have an existing value

Reply:
no, you're right. thanks I just changed it